params_test.go

```diff
@@ -1,43 +1,43 @@
-package fuego
+package fuego_test
 
 import (
+	"strconv"
 	"testing"
 
 	"github.com/stretchr/testify/require"
+
+	"github.com/go-fuego/fuego"
+	"github.com/go-fuego/fuego/option"
 )
 
-func TestParsePathParams(t *testing.T) {
-	require.Equal(t, []string(nil), parsePathParams("/"))
-	require.Equal(t, []string(nil), parsePathParams("/item/"))
-	require.Equal(t, []string{"user"}, parsePathParams("POST /item/{user}"))
-	require.Equal(t, []string{"user"}, parsePathParams("/item/{user}"))
-	require.Equal(t, []string{"user", "id"}, parsePathParams("/item/{user}/{id}"))
-	require.Equal(t, []string{"$"}, parsePathParams("/item/{$}"))
-	require.Equal(t, []string{"user"}, parsePathParams("POST alt.com/item/{user}"))
-}
+func TestParams(t *testing.T) {
+	t.Run("All options", func(t *testing.T) {
+		s := fuego.NewServer()
 
-func BenchmarkParsePathParams(b *testing.B) {
-	b.Run("empty", func(b *testing.B) {
-		for range b.N {
-			parsePathParams("/")
-		}
-	})
+		route := fuego.Get(s, "/test", func(c fuego.ContextNoBody) (string, error) {
+			name := c.QueryParam("name")
+			age := c.QueryParamInt("age")
+			isok := c.QueryParamBool("is_ok")
```

Comment:
nitpicking: I would like to suggest

```suggestion
			isOK := c.QueryParamBool("is_ok")
```